utils/grpcutil.go

```diff
@@ -41,16 +44,25 @@ func Connect(address string) (*grpc.ClientConn, error) {
 				}))
 	}
 
+	// Add retry support.
+	dialOptions = append(dialOptions, grpc.WithUnaryInterceptor(grpc_retry.UnaryClientInterceptor()))
+
 	conn, err := grpc.Dial(address, dialOptions...)
 	if err != nil {
 		return nil, err
 	}
 
-	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
I'm not sure if we really need to wait for a minute. But maybe 10 seconds would be too short. Would need help here.